code/modules/tgui/modules/ert_manager.dm

```diff
@@ -20,16 +20,8 @@
 
 /datum/ui_module/ert_manager/ui_data(mob/user)
 	var/list/data = list()
-	data["str_security_level"] = capitalize(get_security_level())
-	switch(GLOB.security_level)
-		if(SEC_LEVEL_GREEN)
-			data["security_level_color"] = "green"
-		if(SEC_LEVEL_BLUE)
-			data["security_level_color"] = "blue"
-		if(SEC_LEVEL_RED)
-			data["security_level_color"] = "red"
-		else
-			data["security_level_color"] = "purple"
+	data["str_security_level"] = capitalize(SSsecurity_level.get_current_level_as_text())
+	data["security_level_color"] = SSsecurity_level.current_security_level.color
```

Comment:
You need to verify this works on the TGUI end, stuff like `limegreen` and `dodgerblue` likely won't work.

Reply:
Colors are present:D 
![image](https://github.com/ParadiseSS13/Paradise/assets/44334376/d9b334a5-c854-4812-bbb1-0e72a7f880e9)
![image](https://github.com/ParadiseSS13/Paradise/assets/44334376/a322c3e5-1822-4c4a-b8df-da4f00524eec)